modules/@apostrophecms/ui/ui/apos/scss/global/_theme.scss

```diff
@@ -1,9 +1,3 @@
-body {
```

Comment:
This was the primary culprit of this entire PR. This was the only place we were defining the UI type, and it had the side effect of effecting project-cod